contrib/utils/init_embedding.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
```

Comment:
Please add the Apache 2.0 license header (as found in all other source files). 
Could you also state in a separate comment whether you are fine with the Apache 2.0 license for this code? Thank you!